backend/development.ini.sample

```diff
@@ -101,6 +101,9 @@ auth_type = internal
 # ldap_group_enabled = False
 # User auth token validity in seconds (used to interfaces like web calendars)
 user.auth_token.validity = 604800
+# user reset_password token validity (default to 600s -> 10 minutes)
+user.reset_password.validity = 600
```

Comment:
This value has been validated by @lebouquetin ? Some mail provider can be slow, maybe 10 minute is too less.

Reply:
No validation now. I agree 10 minutes is a bit low, 15 minutes maybe ?

Reply:
I was thinking as something like some one hour or two. Take a talk with @PhilippeAccorsi about that.